dependencies.gradle

```diff
@@ -28,6 +28,7 @@ ext.versions = [
 
         junit                  : '4.13.2',
         turbine                : '0.7.0',
+        stately                : '1.2.0'
```

Comment:
Well, we'll see how well upstream likes the inclusion of stately. But opening a PR there is a good place to ask them whether they'd rather have stately or an included implementation of AtomicInt.